client/src/pages/ProfileList/profileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@mui/system';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import useStyle from './useStyles';
+import Avatar from '@mui/material/Avatar';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, Rating, Divider } from '@mui/material';
+function ProfileCard(): JSX.Element {
```

Comment:
Nit: Let's have a line break to separate the imports from the component def

Reply:
Done the changes.